app/src/src/stylesheets/likesAndComments/likesAndComments.jsx

```diff
@@ -0,0 +1,12 @@
+/* Used in CommentsScreen, LikedScreen */
+
+import { StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
```

Comment:
We can use the 'export' keyword infront of 'const' to save lines.

Also, we should give this a more descriptive name than just 'styles'